### PR TITLE
Add job queue link in provision dialog

### PR DESCRIPTION
**Pull request by t-rufang**
You can see the change in this [image](https://microsoft-my.sharepoint.com/:i:/g/personal/rufan_microsoft_com1/Efwwvk4PiMtFn1rzkQZGl6oB9Fi--F2kpLxLEuXz3Mm1Kg?e=aAbHqq)

### CODE REVIEW

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java [hunk 1]
```diff
@@ -36,20 +37,24 @@
...
+import java.io.IOException;
 import java.net.URI;
 import java.util.Collections;
 import java.util.List;
 
-public class AzureSparkServerlessAccount implements ClusterContainer, Comparable<AzureSparkServerlessAccount> {
+public class AzureSparkServerlessAccount implements ClusterContainer, Comparable<AzureSparkServerlessAccount>, ILogger {
     private static final String REST_SEGMENT_SPARK_RESOURCEPOOLS = "/activityTypes/spark/resourcePools";
     private static final String REST_SEGMENT_JOB_LIST = "/Jobs";
+    private static final String REST_SEGMENT_JOB_MANAGEMENT_INFIX = "/#@microsoft.onmicrosoft.com/resource";
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java]: `microsoft.onmicrosoft.com` should be extracted from some fields.

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java [hunk 2]
```diff
@@ -104,6 +109,34 @@ public URI getUri() {
...
     }
 
+    @Nullable
+    public URI getJobManagementURI() {
+        if (getId() == null) {
+            return null;
+        }
+
+        try {
+            String url = AuthMethodManager.getInstance().getAzureManager().getPortalUrl()
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java]: `AuthMethodManager.getInstance().getAzureManager()` may return `null` for non-signed-in user

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessProvisionDialog.java [hunk 3]
```diff
@@ -81,6 +82,7 @@
     protected JLabel storageRootPathLabel;
     protected JComboBox sparkVersionComboBox;
     protected JLabel sparkVersionLabel;
+    private JXHyperLinkWithUri jobQueueHyperLink;
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessProvisionDialog.java]: `@Nullable`

Comment (t-rufang) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/sparkserverless/serverexplore/ui/SparkServerlessProvisionDialog.java]: We can `setURI(null)` to `jobQueueHyperLink`, but the field itself should not be null

---
In reply to: [209217336](https://github.com/Microsoft/azure-tools-for-java/pull/1872#discussion_r209217336) [](ancestors = 209217336)

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java (hunk 1 above)
Comment (t-rufang) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java]: Replaced with tenant ID of the subscription, which also works.

---
In reply to: [209216559](https://github.com/Microsoft/azure-tools-for-java/pull/1872#discussion_r209216559) [](ancestors = 209216559)

**Pull request merged**